### ISSUE TITLE
Fails to load document from .ts file using decorator

### ISSUE THREAD

**Issue by adwd** [ardatan/graphql-toolkit#492]
Hello, I'm using graphql-codegen for my Angular app. I found Angular component with decorator makes codegen fail.
This error come to happen when I update `@graphql-codegen/cli` to `1.13.0` from `1.9.1` . `1.9.1` depends `@graphql-toolkit/*@0.7.4` .

Below is sample code.

```ts
// angular-component.ts
import { Component, Input, Optional, ChangeDetectorRef } from '@angular/core';
import gql from 'graphql-tag';

const query = gql`
  query Users {
    users { name }
  }
`;

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.css']
})
export class AppComponent {
   // this @Input decorator does not affect
  @Input() foo: any;

  // this @Optional decorator causes error
  constructor(@Optional() cd: ChangeDetectorRef) { }
  // constructor() {} // this works
}

```

```js
// index.js
const { loadDocuments } = require('@graphql-toolkit/core');
const { CodeFileLoader } = require('@graphql-toolkit/code-file-loader');

loadDocuments('./angular-component.ts', {
  loaders: [
    new CodeFileLoader()
  ]
}).then(console.log);
```

```sh
$ node index.js
(node:24070) UnhandledPromiseRejectionWarning: SyntaxError: Stage 2 decorators cannot be used to decorate parameters (22:14)
    at Object.raise (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:7013:17)
    at Object.parseBindingList (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:8768:16)
    at Object.parseFunctionParams (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:11386:24)
    at Object.parseFunctionParams (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:6165:11)
    at Object.parseMethod (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:10159:10)
    at Object.pushClassMethod (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:11638:30)
    at Object.pushClassMethod (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:6135:11)
    at Object.parseClassMemberWithIsStatic (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:11555:12)
    at Object.parseClassMemberWithIsStatic (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:6008:11)
    at Object.parseClassMember (/Users/adwd/work/graphql/test-graphql-toolkit/node_modules/@babel/parser/lib/index.js:11497:10)
(node:24070) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:24070) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by ardatan** [comment]
Available in v0.9.8 !!!

**Comment by adwd** [comment]
Thanks!!!
